### PR TITLE
[device] Misc fixes for Arista platforms

### PR DESCRIPTION
**Pull request by Staphylo**
**- What I did**

Fix the i2c bus names for 7050QX-32 and 7050QX-32S after the recent submodule update.
Fix the default minigraph.xml for the 7050QX-32S that prevented it from properly starting.
Allow the image to be booted from another partition which does not affect default behavior.

**- How to verify it**

Boot an image on 7050QX-32S from vfat to verify that the initramfs script is still working.
Starting without minigraph.xml and verifying that syncd does not crash verifies the default minigraph.xml
Verify that the sensors names are properly set up.

**- Description for the changelog**

Miscellaneous fixes for Arista devices.

### CODE REVIEW

File: files/image_config/interfaces/interfaces.j2 [hunk 1]
```diff
@@ -22,20 +22,20 @@ iface lo {{ 'inet' if prefix | ipv4 else 'inet6' }} static
 auto eth0
 {% if MGMT_INTERFACE %}
 {% for (name, prefix) in MGMT_INTERFACE %}
-iface eth0 {{ 'inet' if prefix | ipv4 else 'inet6' }} static
+iface {{ name }} {{ 'inet' if prefix | ipv4 else 'inet6' }} static
```
Comment (lguohan) [inline files/image_config/interfaces/interfaces.j2]: why do you replace eth0 with {{ name }}? there are lots of places that assume the management port name is eth0. is this assumption broken on arista platform?

Comment (yurypm) [reply files/image_config/interfaces/interfaces.j2]: this assumption is not broken.
1) Replace eth0 by '{{ name }}' as it should be
2) this fix allows to have more than one mgmt interfaces: eth0 + XXX + ...

Comment (lguohan) [reply files/image_config/interfaces/interfaces.j2]: can you revert this?

I checked, current the name is Management0, I do not think we should use that.

Comment (Staphylo) [reply files/image_config/interfaces/interfaces.j2]: Sure, I'll remove this commit.

**Pull request merged**